server/app/controllers/service-controller.js

```diff
@@ -10,7 +10,9 @@ const { checkStructure, createUserError } = require('../../utils');
  * @return {Promise.<void>} Nothing
  */
 async function getAllServices(req, res) {
-    const services = await serviceService.getAllServices();
+    const start = req.query.start;
```

Comment:
You must transform date values from timestamp with millisecond to Date (Javascript format). Check barman controller in clement's branch.